package.json

```diff
@@ -112,12 +112,15 @@
     "enzyme-to-json": "^3.4.4",
     "eslint": "^7.19.0",
     "gh-pages": "^3.1.0",
-    "html-webpack-plugin": "^4.5.0",
+    "html-webpack-plugin": "^3.1.0",
```

Comment:
This wasn't used before, downgraded to match what is used in terra-dev-site currently.